packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -94,6 +94,24 @@ const SearchPage = props => {
         </div>
     ) : null;
 
+    const searchResultsHeading = searchTerm ? (
+        <FormattedMessage
+            id={'searchPage.searchTerm'}
+            values={{
+                highlight: chunks => (
+                    <span className={classes.headingHighlight}>{chunks}</span>
+                ),
+                category: searchCategory,
+                term: searchTerm
+            }}
+        />
+    ) : (
+        <FormattedMessage
+            id={'searchPage.searchTermEmpty'}
+            defaultMessage={'Showing all results:'}
```

Comment:
Shouldn't need the default here since the `id` directly correlates to the string translation. You need defaults when there is a template string/replacement in the json.

Reply:
https://github.com/magento/pwa-studio/pull/2685/commits/7a888d7fab8024edb7344f26f40776677e1ed921